### DISCUSSION TITLE
ghcr.io/siderolabs/kubelet:v1.23.5 Service Unavailable

### DISCUSSION THREAD

**Discussion by ilvpozzd** [siderolabs/talos#5429] [General]
![Screenshot from 2022-04-25 13-39-53](https://user-images.githubusercontent.com/81648305/165074722-5bda32da-abf5-46e9-9e4c-15d113418fcd.png)

![image](https://user-images.githubusercontent.com/81648305/165074734-c8e0c8dc-e533-4145-9cd5-95e77c34b9b4.png)

**Comment by sergelogvinov** [comment]
Yep, unfortunately this is github issue. It has outage today... https://www.githubstatus.com/
